src/main/java/com/bgsoftware/superiorskyblock/island/top/SortingTypes.java

```diff
@@ -27,7 +27,7 @@ private static SortingType register(String name, Comparator<Island> comparator,
     }
 
     public static SortingType getDefaultSorting() {
-        return SuperiorSkyblockPlugin.getPlugin().getSettings().getIslandTopOrder();
+        return SortingType.getByName(SuperiorSkyblockPlugin.getPlugin().getSettings().getIslandTopOrder());
```

Comment:
Let's cache the value as well.

Reply:
Wdm by cached ? actually it's like that ?